recipes/pygnome/meta.yaml

```diff
@@ -18,13 +18,14 @@ requirements:
   build:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - numpy                               # [build_platform != target_platform]
+    - numpy
     - cython
     - netCDF4
     - scikit-build-core
     - clang 15.*             # [osx]
     - {{ compiler('c') }}    # [not osx]
     - {{ compiler('cxx') }}  # [not osx]
+    - make
```

Comment:
Should this be [now win] ?

Reply:
Maybe.  I don't think it is bothering the windows build step though, so I was keeping it simple.

Reply:
```suggestion
    - ninja
```